Tests/js/models/assets/ez-usermodel-tests.js

```diff
@@ -14,7 +14,9 @@ YUI.add('ez-usermodel-tests', function (Y) {
             this.serviceMock = new Y.Mock();
             this.serviceLoad = 'loadUser';
             this.rootProperty = "User";
-            this.parsedAttributeNumber = Y.eZ.User.ATTRS_REST_MAP.length + 1; // links
+            this.getFieldDefinitionIdentifiers = 'getFieldDefinitionIdentifiers';
+            this.contentType = new Y.Mock();
+            this.parsedAttributeNumber = Y.eZ.User.ATTRS_REST_MAP.length + 2; // links
```

Comment:
you should update the comment to `// links + avatar` since it's not "+ 1" anymore